### ISSUE TITLE
Live examples

### ISSUE THREAD

**Issue by ghost** [tsers-js/core#8]
It would be nice something like tricicle (Cycle online) but for TSERS or examples at JS Bin. You know, something that let you play with TSERS examples online.
Thank you for TSERS, it really seems an improved cycle.js.

**Comment by milankinen** [comment]
Yeah, that's a nice idea! Perhaps I could add some. At least the hello world example. (Or does anyone volunteer :sunglasses:?)
